### PR TITLE
Bump grafana and prometheus versions

### PR DESCRIPTION
**Pull request by howardjohn**
I have manually spot tested these both and prometheus has automated
testing

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: samples/addons/grafana.yaml [hunk 1]
```diff
@@ -19,10 +19,10 @@ metadata:
   name: grafana
   namespace: istio-system
   labels:
-    helm.sh/chart: grafana-5.0.7
+    helm.sh/chart: grafana-5.1.2
     app.kubernetes.io/name: grafana
     app.kubernetes.io/instance: grafana
-    app.kubernetes.io/version: "6.6.2"
+    app.kubernetes.io/version: "7.0.1"
```
Comment (douglas-reid) [inline samples/addons/grafana.yaml:25]: I think we need to be on 7.0.2 -- there was a security patch release today.

Comment (howardjohn) [reply samples/addons/grafana.yaml]: Good point. I updated this main as a result of that and then didn't actually use that version... I just updated it

**Review by douglas-reid — APPROVED** [review]

**Review by morvencao — APPROVED** [review]

**Pull request merged**